site/specs/numbers.json

```diff
@@ -3287,10 +3276,10 @@
         },
         "responses": {
           "200": {
-            "description": "<p>OK - note has been updated.  No body is returned</p>\n"
+            "description": "OK - note has been updated.  No body is returned\n"
           },
           "400": {
-            "description": "<p>Bad Request - the note has not been updated because the payload is incomplete or in error.  An error payload is provided in the response.</p>\n",
+            "description": "Bad Request - the note has not been updated because the payload is incomplete or in error.  An error payload is provided in the response.\n",
```

Comment:
```suggestion
            "description": "Bad Request - the note has not been updated because the payload is incomplete or in error.  An error payload is provided in the response.",
```